src/AnnotationDialog.js

```diff
@@ -588,31 +520,15 @@ class AnnotationDialog extends EventEmitter {
             }
             dialogEl.appendChild(createSectionEl);
 
-            const createTextArea = document.createElement('textarea');
-            createTextArea.classList.add(constants.CLASS_TEXTAREA);
-            createTextArea.classList.add(constants.CLASS_ANNOTATION_TEXTAREA);
-            createTextArea.placeholder = this.localized.addCommentPlaceholder;
-            createSectionEl.appendChild(createTextArea);
-
-            const createBtnsContainer = document.createElement('div');
-            createBtnsContainer.classList.add(constants.CLASS_BUTTON_CONTAINER);
-            createSectionEl.appendChild(createBtnsContainer);
-
-            const cancelBtn = util.generateBtn(
-                [constants.CLASS_BUTTON, constants.CLASS_ANNOTATION_BUTTON_CANCEL],
-                this.localized.cancelButton,
-                this.localized.cancelButton,
-                constants.DATA_TYPE_CANCEL
-            );
-            createBtnsContainer.appendChild(cancelBtn);
-
-            const postBtn = util.generateBtn(
-                [constants.CLASS_BUTTON, constants.CLASS_BUTTON_PRIMARY, constants.CLASS_ANNOTATION_BUTTON_POST],
-                this.localized.postButton,
-                this.localized.postButton,
-                constants.DATA_TYPE_POST
+            const language = this.locale.substr(0, this.locale.indexOf('-'));
```

Comment:
Doesn't the `Internationalize` hoc parse out the language from the locale already? At least it looks like it does in the ui-elements one

Reply:
oh you're totally right! Removing the duplication 👍